src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultCompare = function defaultCompare(a, b) {
+    return a.toString() > b.toString();
+  };
+
+  [].__proto__.sort2 = function (compareFunction = defaultCompare) {
+    const compare = compareFunction;
```

Comment:
There's no need to reassign the `compareFunction` to a new variable `compare`. You can directly use `compareFunction` within your sort implementation.